index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse!</title>
+    <link rel="stylesheet" href="css/main.css">
```

Comment:
- [OPTIONAL] It always better to specify the path, sometimes it's can happen that your index file does not recognize where your CSS file is exactly located. In order to prevent that, you can add `./` before the CSS file name,  this should be like that: ` <link rel="stylesheet" href="./css/main.css">`